pkg/image/trigger/buildconfigs/buildconfigs.go

```diff
@@ -175,10 +176,20 @@ func (r *buildConfigReactor) ImageChanged(obj runtime.Object, tagRetriever trigg
 			newSource = true
 		}
 
-		// LastTriggeredImageID is an image ref, despite the name
-		if latest == p.LastTriggeredImageID {
-			continue
+		// Determine whether this trigger has fired previously and is recorded in status;
+		// openshift-apiserver updates the status of image change triggers,
+		// including resolving LastTriggeredImageID and LastTriggerTime.
+		pStatus := ocmbuildutil.GetImageChageTriggerStatusForImageChangeTrigger(p, bc)
```

Comment:
```suggestion
		pStatus := ocmbuildutil.GetImageChangeTriggerStatusForImageChangeTrigger(p, bc)
```
s/GetImageChageTriggerStatusForImageChangeTrigger/GetImageChangeTriggerStatusForImageChangeTrigger/ everywhere?

Reply:
thanks